.github/workflows/pull_request-or-main.yml

```diff
@@ -12,22 +12,22 @@ on:
       - main
 
 env:
-  DEFAULT_TARGET_PLATFORM: 'Eclipse-2021-12'
+  DEFAULT_TARGET_PLATFORM: 'Eclipse-2022-12'
 
 jobs:
 
   platform:
     strategy:
       fail-fast: false
       matrix:
-        target-platform: ['Eclipse-4.8', 'Eclipse-2021-09', 'Eclipse-2021-12', 'Eclipse-2022-03']
+        target-platform: ['Eclipse-4.8', 'Eclipse-2022-09', 'Eclipse-2022-12', 'Eclipse-2023-03']
```

Comment:
Please be aware that the full matrix might not work because old target platforms may not run on newer java versions (for example due to the changes of javax.xml in Java 9).

Reply:
thanks, it works for now, only the update site build is failing, because of missing dependecy `org.eclipse.equinox.ds` ...